### ISSUE TITLE
Cannot unpack php-http/httplug-pack

### ISSUE THREAD

**Issue by Nyholm** [symfony/flex#315]
Composer version:  1.6.3 2018-01-31 16:28:17

To reproduce: 
```cli
▶ composer create-project symfony/skeleton testUnpack
▶ cd testUnpack
▶ composer req http debug
▶ composer unpack debug
Unpacked symfony/debug-pack dependencies
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Package operations: 0 installs, 0 updates, 1 removal
  - Removing symfony/debug-pack (v1.0.4)

▶ composer unpack http
Nothing to unpack

▶ composer unpack php-http/httplug-pack
Nothing to unpack
```

The [httplug-pack's composer.json](https://github.com/php-http/httplug-flex-pack) has `"type": "symfony-pack",`.

**Comment by fabpot** [comment]
I understand what's going on. Flex uses one version of the package, and it happens that you didn't have the right Composer type for version 1.0. Fixed by #321

**Comment by Nyholm** [comment]
Thank you
